jproject-fr/easy-eat
# #K600: Added make build-frontend to build only the frontend

Added a command make build-end to make the front work after a restart.
Only building the frontend wasn't enough but if i stopped then restarted the container it worked. 

TO TEST:
 - Shutdown docker
 - If the frontend doesn't shows a 503 then restart the computer
 - run `make build-frontend`
 - The frontend should work
